datahub-web-react/src/app/entityV2/view/builder/ViewBuilderModal.tsx

```diff
@@ -44,43 +35,22 @@ const getTitleText = (mode, urn) => {
 
 export const ViewBuilderModal = ({ mode, urn, initialState, onSubmit, onCancel }: Props) => {
     const [viewBuilderState, setViewBuilderState] = useState<ViewBuilderState>(initialState || DEFAULT_BUILDER_STATE);
+    const [showConfirmationModal, setShowConfirmationModal] = useState(false);
 
     useEffect(() => {
         setViewBuilderState(initialState || DEFAULT_BUILDER_STATE);
     }, [initialState]);
 
-    const confirmClose = () => {
-        Modal.confirm({
-            title: 'Exit View Editor',
-            content: `Are you sure you want to exit View editor? All changes will be lost`,
-            onOk() {
-                onCancel?.();
-            },
-            onCancel() {},
-            okText: 'Yes',
-            maskClosable: true,
-            closable: true,
-        });
-    };
-
     const canSave = viewBuilderState.name && viewBuilderState.viewType && viewBuilderState?.definition?.filter;
     const titleText = getTitleText(mode, urn);
 
     return (
-        <ClickOutside onClickOutside={confirmClose} wrapperClassName="test-builder-modal">
+        <ClickOutside onClickOutside={() => setShowConfirmationModal(true)} wrapperClassName="test-builder-modal">
             <Modal
                 wrapClassName="view-builder-modal"
-                footer={null}
-                title={
-                    <TitleContainer>
-                        <Typography.Text>{titleText}</Typography.Text>
-                    </TitleContainer>
-                }
-                style={modalStyle}
-                bodyStyle={modalBodyStyle}
-                visible
-                width={modalWidth}
-                onCancel={onCancel}
+                buttons={[]}
+                title={titleText}
```

Comment:
according to the meticulous diffs this modal could stand to be a bit wider maybe?

Reply:
yeah look when we add some filters:
<img width="1073" height="805" alt="Screenshot 2025-08-20 at 11 25 36 AM" src="https://github.com/user-attachments/assets/7f9b8610-ff3a-49e0-8bd4-744c5cd8cfd5" />